spring-cloud-starter-stream-processor-python-jython/pom.xml

```diff
@@ -19,15 +19,10 @@
 			<groupId>org.springframework.cloud.stream.app</groupId>
 			<artifactId>python-app-starters-common</artifactId>
 		</dependency>
-		<dependency>
-			<groupId>org.springframework.integration</groupId>
-			<artifactId>spring-integration-java-dsl</artifactId>
-			<version>1.2.1.RELEASE</version>
-		</dependency>
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-groovy</artifactId>
-			<version>4.2.11.RELEASE</version>
+			<version>5.0.3.RELEASE</version>
```

Comment:
Must be `5.1.1` . Although I guess you just can rely on the dependency provided by Boot. So, just omit version here!